src/test/smokeTest.node.ts

```diff
@@ -28,25 +27,6 @@ class TestRunner {
 
         await this.launchTest(env);
     }
-    private async enableLanguageServer(enable: boolean) {
-        // When running smoke tests, we won't have access to unbundled files.
-        const settings = `{ "python.languageServer": ${enable ? '"Microsoft"' : '"Jedi"'} }`;
```

Comment:
This is very old code and the setting is wrong, there's no value `Microsoft` in the enum anymore.